docs/DeveloperGuide.md

```diff
@@ -928,29 +928,76 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
 - `remark 100000 r/Birthday on 31 July.`.
 - `remark 1 r/Birthday on 31 July r/`.
 
-### Deleting a tag
+### Adding one or more tags to a person
 
-1. Deleting a tag while all persons are being shown
+#### Successful addition of one or more tags to a person
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
+
+**Example Test Case**: `addtag 1 t/PRUSafe365`.
+
+**Expected Result:**
+1. `PRUSafe365` is added to the first contact on the list.
+2. *Person List* shows all persons in FINDvisor. 
+3. *Command Result Box* outputs a successful execution message with the details of the contact with the added tags.
+4. *Command Box* is cleared.
 
-   1. Test case: `delete 1 t/validTag`<br>
-      Expected: First contact is selected from the list. Tags of the selected contact will be checked.
-      If validTag exisits in the person's tags, it will be removed. Successful output will be showned in the status message.
-      If validTag does not exisit in the person's tags, failure output will be showned in the status message.
-      Timestamp in the status bar is updated.
+**Alternative Test Cases:**
+- `addtag 1 t/PRUSafe365 t/PRUGrowth`.
+- `addtag 5 t/PRUActiveCash`.
 
-   1. Test case: `delete 0 t/validTag`<br>
-      Expected: No tag is deleted as 0 is not a valid index for person. Error details shown in the status message. Status bar remains the same.
+#### Invalid index, tag or command format supplied when adding one or more tags to a person
 
-   1. Test case: `delete 1 t/invalidTag`<br>
-      Expected: No tag is deleted as the tag is invalid. Error details shown in the status message. Status bar remains the same.
+**Prerequisites:**
+1. List all persons using the `list` command. Multiple persons are in the list (less than 100000).
```

Comment:
I think we should change this to `less than 100`? The cases for this should change as well.